rector-upgrade-twill-config.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+use A17\Twill\Rector\LegacyTableConfig;
+use Rector\Config\RectorConfig;
+
+return static function (RectorConfig $config): void {
+    $config->paths([
+        getcwd() . '/config/twill.php',
+    ]);
+
+    $config->ruleWithConfiguration(LegacyTableConfig::class, [
+        'path' => getcwd(),
+        'tables' => [
```

Comment:
I got a bit confused about this because it does it right, but there is no 'tables' prefix. Just to realise it is just the key used for the rule config.